packaging/helm/trento-server/charts/trento-web/templates/deployment.yaml

```diff
@@ -55,6 +55,8 @@ spec:
             - name: TRENTO_COLLECTOR_PORT
               value: "{{ .Values.collectorService.port }}"
             - name: TRENTO_GRAFANA_URL
+              value: /grafana
```

Comment:
> Hey @nelsonkopliku , I don't really understand the new implementation. It is strange that now we need to provide `/grafana` as `GRAFANA_URL`, when this value is not even an url. Anyway, if @fabriziosestito approves it, as he did the initial grafana implementation, I'm fine with it

@arbulu89 apart from the naming, another parameter here is needed because the "api url" is using the service in the k8s cluster, while the other url is the one that we expose to the users to show the iframe.

Reply:
Ok, I'm wondering if we need to ask this to the user, or if `/grafana` will be the value always.
@fabriziosestito Where is this value coming from? Do we configure this in our grafana deployment?

Reply:
I see.
what if we rename `url` to `embeddable_url`? Would that help?

Reply:
`/grafana` is what we configured in the Ingress: https://github.com/trento-project/trento/blob/adb41196469f19ae5aad56e3f6124534efdfd86e/packaging/helm/trento-server/values.yaml#L58

Reply:
> I see. what if we rename `url` to `embeddable_url`? Would that help?

What about `PublicURL`?
`/grafana` is a relative URL so IMHO it is acceptable keeping the url suffix in the parameter.

Reply:
I agree, a URL can be either absolute or relative.
If `PublicURL` helps I'll change it.

@arbulu89 ok with you?

Reply:
changed :rocket:
thanks for the feedback!